plugin/code_actions.py

```diff
@@ -256,7 +256,8 @@ def _handle_response(self, responses: CodeActionsByConfigName) -> None:
                 for code_action in code_actions:
                     run_code_action_or_command(self._view, config_name, code_action)
         if document_version != self._view.change_count():
-            self._request_code_actions()
+            # Give on_text_changed_async a chance to trigger.
+            sublime.set_timeout_async(self._request_code_actions_async)
```

Comment:
I'm not sure if this would solve the problem of on_text_changed_async being "too late". I'm guessing this should work because `run_code_action_or_command` does a bunch of edits, and then the on_text_changed_async is scheduled. And then this line schedules _request_code_actions_async after it. Hopefully :)

Reply:
Yes, I would think that's a pretty safe assumption.